### ISSUE TITLE
Don't know how to build task 'archive:development'

### ISSUE THREAD

**Issue by defvol** [HipByte/motion-testflight#6]
When I `rake testflight` I get:

```
rake aborted!
Don't know how to build task 'archive:development'

Tasks: TOP => testflight => testflight:submit
(See full trace by running task with --trace)
```

RubyMotion 1.24 removes archive:development in favor of mode=development

= RubyMotion 1.24 =
- Added to the build system the `mode' option which can be used to specify the build mode, either `development' (default) or `release'. You can now test your app in release mode by using `rake mode=release' for simulator and `rake device mode=release' for device.
- Fixed the build system not to execute the user-defined application setup block twice, which is no longer necessary with the new `mode' option.
- **Removed the `archive:development'** and `archive:release' tasks which are no longer necessary with the new `mode' option.
- Added the `archive:distribution' task which builds an .ipa with distribution settings (certificate, default entitlements, symbols stripped).

**Comment by lrz** [comment]
Oops!

**Comment by defvol** [comment]
line 87 :)

testing ...

**Comment by lrz** [comment]
Sounds like the following should work:

```
index 0d72d01..8ec57b3 100644
--- a/lib/motion/project/testflight.rb
+++ b/lib/motion/project/testflight.rb
@@ -83,8 +83,8 @@ module Motion; module Project; class Config
 end; end; end

 namespace 'testflight' do
-  desc "Submit a development archive to TestFlight"
-  task :submit => 'archive:development' do
+  desc "Submit an archive to TestFlight"
+  task :submit => 'archive' do
     # Retrieve configuration settings.
     prefs = App.config.testflight
     App.fail "A value for app.testflight.api_token is mandatory" unless prefs.api_token
```

**Comment by defvol** [comment]
what about archive:distribution ?

**Comment by defvol** [comment]
should we check for versions < 1.24 ?

**Comment by lrz** [comment]
I think :archive is better as you may want to submit a development build. The user can set the `mode` option to `release` if a release build is needed. archive:distribution will not work as it will be signed with the Distribution certificate (which only works for App Store submissions afaik).

**Comment by lrz** [comment]
I just fixed this and submitted motion-testflight 1.2. Thanks for the report!

**Comment by defvol** [comment]
:+1:
